.changeset/two-lamps-tell.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
+---
+
+Fixes the terminal HMR path
```

Comment:
Astro users will see this in the changelog, and they may not have full context about what happened.
"Fixed an issue where the dev server logged the full file path on updates."